.ci-scripts/library

```diff
@@ -1,6 +1,6 @@
 #!/bin/echo Warning: this file should be sourced!
 
-DOCKER_IMAGE="cita/cita-build:ubuntu-18.04-20181009"
+DOCKER_IMAGE="cita/cita-build:ubuntu-18.04-20190107"
```

Comment:
The changes for new docker: [cita/cita-build:ubuntu-18.04-20190107](https://github.com/cryptape/cita-build/commit/53ef2667b037ed54c5389185d1ed06088e33fe2f).